src/index.js

```diff
@@ -1,2 +1,46 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+
+function createElement(tagName, props = {}, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+
+  children.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+    element.appendChild(document.createTextNode(child));
+  });
+  return element;
+}
+
+function render(valueRender) {
+  const element = (
+    <div id="hello" className="greeting">
+      <p>hello World</p>
+      <p>
+        <button
+          id="button"
+          type="button"
+          onClick={() => render(`Click me! (${parseInt((document.getElementById('button').innerText.split(' ')[2]).split('').slice(1, -1), 10) + 1})`)}
+        >
+          {valueRender}
+        </button>
+      </p>
+      <p>
+        {[1, 2, 3].map((i) => (
+          <button type="button" onClick={() => render(`Click me! (${i})`)}>
+            {i}
+          </button>
```

Comment:
```suggestion
          <button type="button" onClick={() => render(i)}>
            Click me! {i}
          </button>
```

처럼 오히려 매개변수는 간단하게 값만 넘겨주고 보여주는 text 등은 표여지는 곳에서 표현하는게 좋을 것 같아요!